<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXWidget</class>
 <widget class="QWidget" name="CXWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mHostEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mPortEdit">
       <property name="text">
        <string>7755</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mStartButton">
       <property name="text">
        <string>Запустить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="mSendEdit">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mSendButton">
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Входящее сообщение</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="mReceiveView">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mClearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
